### PR TITLE
Fixed many typos

### PR DESCRIPTION
**Pull request by polluks2**

### CODE REVIEW

**Comment by WayneParham** [comment]
Good catches, thanks!

**Comment by polluks** [comment]
@WayneParham It was a Codespell job.

File: libsrc/cbm/read.s [hunk 1]
```diff
@@ -135,7 +135,7 @@ eof:    lda     #0
 
 devnotpresent:
         lda     #ENODEV
-        jmp     __directerrno   ; Sets _errno, clears _oserror, returns -1
+        .byte   $2C             ; Skip next opcode
```
Comment (mrdudz) [inline libsrc/cbm/read.s:138]: this shouldnt be part of this PR

File: libsrc/cbm/write.s [hunk 2]
```diff
@@ -106,7 +106,7 @@ devnotpresent2:
         pla
 devnotpresent:
         lda     #ENODEV
-        jmp     __directerrno   ; Sets _errno, clears _oserror, returns -1
+        .byte   $2C             ; Skip next opcode
```
Comment (mrdudz) [inline libsrc/cbm/write.s:109]: this shouldnt be part of this PR

**Review by mrdudz — CHANGES_REQUESTED** [review]
please remove the code changes from this PR

**Review by polluks — APPROVED** [review]
done

**Pull request merged**